QRHunter/app/src/main/java/com/example/qrhunter/ui/scan/ScanViewModel.java

```diff
@@ -42,26 +46,26 @@ public void scanQRCode(String qrCodeContent) {
      *
      * @param playerId The player that's scanning the qr code
      */
-    public void completeScan(String playerId) {
+    public void completeScan(String playerId, byte[] savedPhoto) {
         ArrayList<Location> locations = new ArrayList<>();
         ArrayList<String> photos = new ArrayList<>();
 
+        FirebaseFirestore db = FirebaseFirestore.getInstance();
+        DocumentReference ref = db.collection("qrCodes").document();
+        String qrCodeId = ref.getId();
+
```

Comment:
Is it possible to move this to `addQRCodeToPlayer()` in `qrCodeRepository`? Would be ideal to not call Firestore in ViewModels

Reply:
Will do that